Aflores plymouth (#234)
osisoft/Edge-Data-Store-Docs: content/diagnostics/diagnostics.md

# Diagnostics configuration

> Edge Data Store and its components produce performance and system data for diagnostic purposes which is stored locally in the Storage component, and may be queried locally or egressed to PI Web API endpoints or the AVEVA Data Hub or both. Diagnostic data is always produced and saved; it cannot be disabled.

Reviewer: ```suggestion
Edge Data Store and its components produce performance and system data for diagnostic purposes and store that data locally in the Storage component. You may query it locally or egress it to PI Web API endpoints, the AVEVA Data Hub, or both. Diagnostic data is always produced and saved; you cannot disable it.
```

> Diagnostic data is stored within the `diagnostics` namespace in Edge Storage. Local access to this data is available through SDS. For more information, see [Read data](xref:sdsReadingData).

Reviewer: ```suggestion
Edge Storage stores diagnostic data within the `diagnostics` namespace. Local access to this data is available through SDS. For more information, see [Read data](xref:sdsReadingData).
```

### Error rate

> The `Diagnostics.Adapter.ErrorRate` dynamic type includes these values, and are logged in a stream with the id `{DeviceName}.{componentid}.ErrorRate`.

Reviewer: ```suggestion
The `Diagnostics.Adapter.ErrorRate` dynamic type includes these values, and are logged in a stream with the Id `{DeviceName}.{componentid}.ErrorRate`.
```
